src/library/scala/collection/package.scala

```diff
@@ -40,39 +40,6 @@ package object collection extends LowPriority {
   @deprecated("Gen* collection types have been removed", "2.13.0")
   val GenMap = Map
 
-  import scala.language.implicitConversions
-  // ------------------ Decorators to add collection ops to existing types -----------------------
-
-  /** Decorator to add collection operations to strings. */
-  def stringToStringOps(s: String): StringOps = new StringOps(s)
-
-  /** Decorator to add collection operations to arrays. */
-  def arrayToArrayOps[A](as: Array[A]): ArrayOps[A] = new ArrayOps[A](as)
```

Comment:
We still need the two conversions above

Reply:
Both are also in `Predef`: [here](https://github.com/scala/scala/blob/v2.13.0-M4-pre-20d3c21/src/library/scala/Predef.scala#L354) and [here](https://github.com/scala/scala/blob/v2.13.0-M4-pre-20d3c21/src/library/scala/Predef.scala#L399-L411). The ones for `ArrayOps` look like they could be a lot simpler now, but the comment says the overload are still necessay...

Reply:
OK, thanks for checking! YMMV but I prefer having them in `scala.collection` instead of `scala.Predef`, because they will be automatically imported if you use any type that is related with the collections even if you compile with `-Yno-predef`.

Reply:
The `StringOps` and `ArrayOps` implicit conversions will have to be in `Predef`, I think. `String` and `Array` are not in the `scala.collection` namespace and we can't (without compiler hacks) give them a companion object. So when calling `"foo".tail` or `Array(1,2,3).head` the only way to automatically have these conversions in implicit scope is by having them imported from `scala._` or `scala.Predef._`.

Reply:
The `Wrapped*` conversions could probably go in `scala.collection`, though I'm never very confident about the "enclosing objects" part of the implicit scope...
However I'm not even sure if you should have _implicit_ conversions for `WrappedString` and `WrappedArray`. But that's probably a different issue/pull request.

Reply:
We could leave the `WrappedXxx` conversions in `scala.collection` if we can check that there will be no ambiguities with the `XxxOps` ones. These conversions are useful to use `Array` or `String` values in a context where a collection (e.g. `Iterable`) is expected. You think we should make it more explicit?

Reply:
> These conversions are useful to use Array or String values in a context where a collection is expected.
Oh riiight, I forgot about that part. There are these very rare occasions when I want to write `WrappedArray(array)` and have to use the implicit conversion or that weird `WrappedArray.make` method instead...
But no, we should keep the implicit conversions as well then.

Reply:
We have all these conversions in `Predef`. We can't put them into the collections package because they need to be prioritized correctly against the conversions to `ArrayOps` and `StringOps`.